engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -171,6 +172,20 @@
 	 */
 	HibernateValidatorConfiguration allowOverridingMethodAlterParameterConstraint(boolean allow);
 
+
+	/**
+	 * Registers custom property accessor which allows extracting bean properties with different naming convention than
+	 * standard Java Beans (eg. {@code foo()} vs {@code getFoo()}).
+	 *
+	 * @param selector custom property selector
+	 *
+	 * @return {@code this} following the chaining method pattern
+	 *
+	 * @since 6.0
```

Comment:
It's an interesting question really.

Strictly speaking, it'd be 6.1 as it's a new feature. But then, the cadence of minor releases has been slow for us traditionally, so should we wait with shipping this for 6 months (arbitrary number just for the sake of discussing it)?

Or should we move to a Chrome/Firefox style and simply release minor versions as soon as there's one new feature, in which case we may arrive at HV 6.42 quickly? Or should we allow ourselves to put out new features like this in micro-versions, too? It's an open question which we probably should take to hibernate-dev, I haven't made up my mind. Only that I wouldn't like holding back finished features for a long time.